service/controller.go

```diff
@@ -618,8 +620,16 @@ func (s *service) CreateVolume(
 							log.Debugf("Error while adding dummy localhost entry to export '%d'", exportID)
 						}
 					}
+					// return the createVolume response with actual array volume name
+					exportPath := path
+					if export.Paths != nil {
+						exportPath = (*export.Paths)[0]
```

Comment:
Is there a possibility of an empty slice here and panic? We should check the length.